tests/ergo_run_test.go

```diff
@@ -80,27 +68,29 @@ func TestIntegration(t *testing.T) {
 			}
 		}
 	})
+}
+func TestShowUrlForName(t *testing.T){ 
 
 	t.Run("it shows the url for a given name", func(tt *testing.T) {
 		appsOutput := map[string]string{
-			"foo":                "http://localhost:3000",
-			"bla":                "http://localhost:5000",
-			"withspaces":         "http://localhost:8080",
-			"one.domain":         "http://localhost:8081",
-			"two.domain":         "http://localhost:8082",
-			"redis://redislocal": "redis://localhost:6543",
+			"foo":                "http://foo.dev",
+			"bla":                "http://bla.dev",
+			"withspaces":         "http://withspaces.dev",
+			"one.domain":         "http://one.domain.dev",
+			"two.domain":         "http://two.domain.dev",
+			"redis://redislocal": "http://redis://redislocal.dev",
```

Comment:
Actually, here we do expect `redis://redislocal.dev`

Reply:
if it is a "test bug" we can open a new issue for a fix

Reply:
I have noticed this too but this is not a test bug, this is the actual value returned by the function.
The function should be modified to check for protocol definition prefix.
I will make a pull request for that and then I will it fix here to.